doc/en/components/data-grid.md

```diff
@@ -10,6 +10,14 @@ The $ProductName$ Data Table / Data Grid is a tabular $Platform$ grid component
 
 ## $Platform$ Data Grid Example
 
+This demo implements some of the features that are available in the Grid:
+
+- Filtering
+- Grouping
+- Pin/Unpin columns
+- Reposition columns
+- Sorting
+- Summaries
```

Comment:
adding this list above the samples will force customer to scroll to see our sample. Jason said many times that the first sample in a topic must be fully/partially visible without any scrolling. You should list  these features in one line instead of a list

Reply:
see commit 2